docs/api/options.md

```diff
@@ -34,6 +34,9 @@
 - `loaders`: Object. See [defineLoaders](#appgraphqlextendschemaschema-appgraphqldefineresolversresolvers-and-appgraphqldefineloadersloaders) for more
   details.
 - `schemaTransforms`: Array of schema-transformation functions. Accept a schema as an argument and return a schema.
+- `graphql`: Object. Override options for graphql function that Mercurius utilizes.
+  - `parseOptions`: Object. [GraphQL's parse function options](https://github.com/graphql/graphql-js/blob/main/src/language/parser.ts#L77-L133)
+  - `validateOptions`: Object. [GraphQL's validate function options](https://github.com/graphql/graphql-js/blob/main/src/validation/validate.ts#L44)
```

Comment:
I would avoid deep links to source code, as it can change any time. The alternative would be to use permalinks, but I'm not sure that's a good option either. Is there official documentation that we can point to instead?

Reply:
The published graphql docs do not contain option properties sadly. I could update them to a specific tag if that works for you?

Reply:
not many compelling options then. I'm not sure. if we point at master, those links may refer to wrong lines when those files change. if we use permalinks, the reference will keep existing but the real code might have changed. not sure what's best really. shall we link to the files without any line number perhaps? that would give us some confidence that those files will keep existing and leave it to the reader to figure out where to find the types

Reply:
We could just enumerate the literal properties, rather than linking off. It does mean our future selfs will need to go back to the docs and confirm when things have been added/removed.

Not sure there is a good option, other than pushing for these functions to be documented more fully in GraphQL library.

Happy to change to without line numbers if that makes anyone feel more comfortable.

Reply:
No line numbers is the compromise I'd go with, but happy for somebody else to chime in too

Reply:
Done! :)